docs/docs/0.4.0/documentation/overview.md

```diff
@@ -63,6 +63,22 @@ You can learn more about sidecar injection in the section on [Dataplanes](./dps-
 <img src="/images/docs/0.2.0/diagram-08.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
+### Specify Mesh for Pods or Dataplanes
+
+When deploying services in Kubernetes, you can determine which Mesh you want the service to be be in by using the `kuma.io/mesh: $MESH_NAME` annotation. This annotation would be applied to a deployment like so:
```

Comment:
@devadvocado this is also possible to do in a `Namespace` correct? If yes, this description is not accurate.

Reply:
No, this is only possible on Deployment